content/NavCoin Core/How to move from NavPi to Raspberry Pi 4 and CM4.en.md

```diff
@@ -0,0 +1,182 @@
+---
+title: How to move from NavPi to Raspberry Pi 4 and CM4 for staking
+date: "2021-05-31"
+order: "000"
+---
+
+
+# Moving from NavPi to headless Raspberry Pi Compute Module 4 for staking
+
+Managing the NavPi through its GUI made managing and staking Navcoin a very simple process. Wanting to use the latest navcore 6.X and moving to a more up-to-date and powerful Raspberry Pi platform made staking more commandline heavy due to the missing GUI component when wanting to stake on a Pi remotely. 
+
+Below is a quick and dirty how to on how to move from a NavPi to a Raspberry Pi Compute Module 4 (CM4) and the latest version of navcoin 6.x to stake Navcoins headless on the Pi.
+
+**Disclaimer**: This how to was made to work with a lot of googeling and asking a ton of stupind questions to the patient Discord members of the Navcoin #navpi-support channel. It may not be the 'only' or 'best' way BUT this is how I was able to get staking working for me.
+
+**Special Thanks to `@salmonskinroll` and `@aguycalled` as well as `u/Zane_TLI` on Reddit for this [post](https://www.reddit.com/r/NavCoin/comments/l8jdcr/staking_with_navcoind_getstakinginfo_return>)**
+
+This how to is using the CM4 [CM4104032](https://www.buyapi.ca/product/raspberry-pi-compute-module-4-wireless-4gb-32gb-cm4104032/).
+
+1. Assumption is you have a running CM4 (how to can be found [here](https://www.youtube.com/watch?reload=9&v=jp_mF1RknU4)) and ssh access to it
```

Comment:
4gb model is required and 8gb even better.

Reply:
will add